aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -71,6 +71,23 @@ Tensor& div_(Tensor& self, const Tensor& other) {
   return native::div_out(self, self, other);
 }
 
+Tensor truncate(const Tensor& tensor) {
+  if (tensor.is_floating_point()) {
+    return tensor.trunc();
+  }
+  return tensor;
```

Comment:
What effect would this be for a complex tensor?

Reply:
I'm not sure. This PR is meant to preserve existing behavior and register it natively. Improving behavior is out of scope.

Reply:
I saw the deleted Python code. Makes sense. Thanks!